package.json

```diff
@@ -97,6 +97,8 @@
     "@turf/bbox": "^6.0.1",
     "@turf/boolean-within": "^6.0.1",
     "@turf/helpers": "^6.1.4",
+    "@turf/transform-translate": "^5.1.5",
+    "@turf/turf": "^5.1.6",
```

Comment:
I think this is adding a ton of unneeded dependencies (see yarn.lock). I think there’s a sub package for this stuff you can use instead?

Reply:
ah good point! I had initially installed it all because I was playing around and wasn't 100% sure what to use. I'll clean that up in the next push

Reply:
![image](https://user-images.githubusercontent.com/33266041/91220595-421d6f00-e6d1-11ea-9d10-3c65570583a9.png)
Couldn't figure out how to get it working with just the transform-translate package. After some research on Turf's issues I believe is that it's a webpack issue that hasn't been resolved. I found similar problems with Webpack 4 and I'm guessing the transformation package hasn't been updated https://github.com/Turfjs/turf/issues/1396 https://github.com/Turfjs/turf/issues/1383

Will leave an issue on Turf's page.

Reply:
Did you change your import after changing the package?

Reply:
I didn't do it exactly as you did. I tried a more general form where I imported all the categories of libraries as I wanted to use the other transform functions in the future. My code in issue below:

https://github.com/Turfjs/turf/issues/1937

![image](https://user-images.githubusercontent.com/33266041/91367797-42deff80-e7bc-11ea-99ed-1baa9f96be05.png)

However, doing it your way gave an interesting different bug.

Reply:
@chrisgervang Please double check to see if the package.json is the way you want it now. Removed the main @turf/turf and added only the needed libraries